src/library/Library.js

```diff
@@ -1,19 +1,65 @@
 import React from 'react';
-import { SafeAreaView, Text, StyleSheet } from 'react-native';
+import {
+  SafeAreaView,
+  Text,
+  StyleSheet,
+  ScrollView,
+  TouchableOpacity
+} from 'react-native';
 import Header from '../Components/Header';
+import FilterGraphic from '../../assets/filter-outline.svg';
+import LessonPlanButton from './components/LessonPlanButton';
 
 const Library = ({ navigation }) => {
   return (
     <SafeAreaView style={styles.container}>
       <Header />
-      <Text>Library of user's lesson plans here!</Text>
+      <SafeAreaView style={styles.inlineTitle}>
+        <Text style={styles.title}>Lesson Plans</Text>
+        <TouchableOpacity>
+          <FilterGraphic height={25} width={25} style={styles.filterButton} />
+        </TouchableOpacity>
+      </SafeAreaView>
+      <ScrollView>
+        <SafeAreaView style={styles.content}>
+          <LessonPlanButton name={'Lesson Plan A'} style={styles.lessonPlan} />
+          <LessonPlanButton name={'Lesson Plan B'} style={styles.lessonPlan} />
+          <LessonPlanButton name={'Lesson Plan C'} style={styles.lessonPlan} />
+          <LessonPlanButton name={'Lesson Plan D'} style={styles.lessonPlan} />
+        </SafeAreaView>
+      </ScrollView>
     </SafeAreaView>
   );
 };
 
 const styles = StyleSheet.create({
   container: {
-    height: '100%'
+    height: '100%',
+    backgroundColor: '#FFFAF5'
+  },
+  title: {
+    fontFamily: 'Poppins-Bold',
+    fontSize: 28,
+    letterSpacing: 0.3,
+    color: '#453E3D'
+  },
+  inlineTitle: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    flexDirection: 'row',
+    marginHorizontal: 30,
```

Comment:
hey i notice there aren't any units here but looking at the documentation didn't yield any info on what exactly the value would be treated as; some people said that some browsers take no units to be pixels. just to be safe, could you change this to %?

Reply:
React treats all numerical style values as px values if there is no unit. I'm not sure if it makes sense to convert this to a %  though -- shouldn't the margins stay the same regardless of the width / height of the device?

Reply:
that's a good point, i'll merge this for now and if any issues come up later re: compatibility we'll revisit this